build.gradle

```diff
@@ -380,6 +380,15 @@ subprojects {
     }
   }
 
+  cleanTest {
+    subprojects.each {
+      delete "${it.buildDir}/test-results/unitTest"
+      delete "${it.buildDir}/reports/tests/unitTest"
+      delete "${it.buildDir}/test-results/integrationTest"
+      delete "${it.buildDir}/reports/tests/integrationTest"
+    }
+  }
+
```

Comment:
Is it useful for unit and integration test to use separate output dirs? Or should they all write to the same output dir as the tests command?

Reply:
> Is it useful for unit and integration test to use separate output dirs? Or should they all write to the same output dir as the tests command?

We need to combine the reports if unit and integration test use the same output dir. Otherwise, the following command can generate only one report.
```./gradlew cleanTest xxx:unitTest xxx:integrationTest```
Also, changing the output dir of unit/integration may introduce some compatibility issue. It seems to me removing the output dirs is simpler.

Reply:
How do we know that the hardcoded directories here are all correct? Is there a way to get that data from gradle instead of hardcoding?

Reply:
You are right. will copy that.